<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNuevaResolucion</class>
 <widget class="QWidget" name="dlgNuevaResolucion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>433</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nueva resolución</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../icons.qrc">
    <normaloff>:/icons/icons/resolution.png</normaloff>:/icons/icons/resolution.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Epígrafe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtEpigrafe"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabResolucion">
      <attribute name="title">
       <string>Resolución</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QTextEdit" name="txtResolucion">
           <property name="toolTip">
            <string>Resolución</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTextEdit" name="txtResolucionTraduccion">
           <property name="toolTip">
            <string>Traducción</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabResumen">
      <attribute name="title">
       <string>Resumen</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QTextEdit" name="txtResolucionResumen">
         <property name="toolTip">
          <string>Resumen</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabNotas">
      <attribute name="title">
       <string>Notas</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QTextEdit" name="txtResolucionNotas"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabOtros">
      <attribute name="title">
       <string>Otros</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QCheckBox" name="chEntendida">
             <property name="text">
              <string>Entendida</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="chVolverMirar">
             <property name="text">
              <string>Volver a mirar</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QCheckBox" name="chTradudida">
             <property name="text">
              <string>Traducida</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QCheckBox" name="chRazonada">
             <property name="text">
              <string>Razonada</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Interesante</string>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QSpinBox" name="spInteresante"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>241</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Provincia</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="myQLineEdit" name="txtProvincia"/>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="btQuitarProvincia">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Capítulo</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="myQLineEdit" name="txtCapitulo"/>
           </item>
           <item row="1" column="2">
            <widget class="QPushButton" name="btQuitarCapitulo">
             <property name="text">
              <string>-</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Páginas</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="txtPaginas"/>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Verbos</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_10">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QListView" name="twVerbs"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Verbo</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtVerbo"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QPushButton" name="btAnadirVerbo">
                 <property name="text">
                  <string>Añadir</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btQuitarVerbo">
                 <property name="text">
                  <string>Borrar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_3">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Expresiones</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QListView" name="twExpresiones"/>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Expresión</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="txtExpresion"/>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_11">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_7">
               <item>
                <widget class="QPushButton" name="btAnadirExpresion">
                 <property name="text">
                  <string>Añadir</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="btQuitarExpresion">
                 <property name="text">
                  <string>Borrar</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <spacer name="horizontalSpacer_4">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_4">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btDetalles">
       <property name="text">
        <string>&amp;Detalles</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btTemas">
       <property name="text">
        <string>&amp;Temas</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>&amp;Cancelar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">widgets/myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtEpigrafe</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>txtResolucion</tabstop>
  <tabstop>txtResolucionTraduccion</tabstop>
  <tabstop>txtResolucionResumen</tabstop>
  <tabstop>txtResolucionNotas</tabstop>
  <tabstop>chEntendida</tabstop>
  <tabstop>chVolverMirar</tabstop>
  <tabstop>chTradudida</tabstop>
  <tabstop>spInteresante</tabstop>
  <tabstop>txtProvincia</tabstop>
  <tabstop>btQuitarProvincia</tabstop>
  <tabstop>txtCapitulo</tabstop>
  <tabstop>btQuitarCapitulo</tabstop>
  <tabstop>btDetalles</tabstop>
  <tabstop>btTemas</tabstop>
  <tabstop>btCancelar</tabstop>
  <tabstop>btOK</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
